common/locales/en/character.json

```diff
@@ -54,6 +54,10 @@
     "costume": "Costume",
     "costumeText": "If you prefer the look of other gear to what you have equipped, check the \"Use Costume\" box to visually don a costume while wearing your battle gear underneath.",
     "useCostume": "Use Costume",
+    "useCostumeInfo1": "Click \"Use Costume\" to equip items to your avatar without affecting the stats from your Battle Gear! This means that you can equip for the best stats on the left, and dress up your avatar with your equipment on the right.",
+    "useCostumeInfo2": "(show more)",
+    "useCostumeInfo3": "(show less)",
```

Comment:
The show more and show less texts should probably be general purpose strings instead of namespaced to useCostumeInfo